sensors/imu/lsm9dsxx.c

```diff
@@ -74,7 +74,8 @@
 
 /* conversions */
 #define GYR2000DPS_CONV_MRAD 1.221730475F /* convert gyroscope value (at scale 2000DPS) to mrad/s */
-#define ACC8G_CONV_MS2       0.244F       /* convert accelerations (at 8G scale) [m/s^2] */
+#define ACC8G_CONV_MG        0.244F       /* convert accelerations (at 8G scale) [m/s^2] */
+#define MG_CONV_MMS2         9.80665F     /* convert milli G to [mm/s^2] */
```

Comment:
In our coding convention, we use lowercase: 
```suggestion
#define MG_CONV_MMS2         9.80665f     /* convert milli G to [mm/s^2] */
```